File: JewelBox VB.NET/JewelBox2004_vs2002/modMain.vb
Module modMain

    Public RowsAffected, index As Integer
    Public numVals, numRows, x As Integer
    Public te1, te2, te3 As String
    Public strConn As String = "Provider = SQLOLEDB;data source=DREAMWORKS;initial catalog=JewelBox;persist security info=False;user id=admin;password = velocity;workstation id=DREAMWORKS;packet size=4096"
    Public repMod As Integer
    Public mytime As String
    Public loguser As String
    Public logpass As String
    Public logsuccess As Boolean
    Public sql As String
    Public adapt As New SqlClient.SqlDataAdapter()
    Public conn As New SqlClient.SqlConnection()
    Public comm As New SqlClient.SqlCommand()
    Public rslogin As New Data.DataSet()
    Public connString As String
    
    Public Sub ConnectDB()
        connString = "initial catalog=JewelBox;password=delta123;persist security info=True;user id=sa;workstation id=GOLDENTIGER;packet size=4096"
        conn = New SqlClient.SqlConnection(connString)
        comm = New SqlClient.SqlCommand(connString, conn)
        comm.CommandType = CommandType.Text
        Try
            conn.Open()
            Debug.WriteLine("Connection Success")
        Catch SqlExcep As SqlClient.SqlException
            Debug.WriteLine(SqlExcep.Message)
        End Try
    End Sub
    Public Sub commitDB()
        conn.Close()
        conn.Dispose()
        conn = Nothing
        Call ConnectDB()
    End Sub

    Public Sub closeDB()

    End Sub

    Public Sub Main()
        Dim frm As New frmSplash()
        frm.ShowDialog()
        ConnectDB()
    End Sub

    Public Function encrypt_pass(ByVal pass As String) As String
        Dim pass1(40) As String
        Dim ascii(40) As String
        Dim pass2 As String
        Dim lenp As Integer
        Dim i As Integer
        lenp = Len(pass)
        i = 0
        While i < lenp
            i = i + 1
            pass1(i) = Mid(pass, i, 1)
            ascii(i) = Asc(pass1(i))
            ascii(i) = ascii(i) + (i + (i - 4))
            pass2 = pass2 & Chr(ascii(i))
        End While
        encrypt_pass = pass2
    End Function
    Public Function decrypt_pass(ByVal pass As String) As String
        Dim pass1(40) As String
        Dim ascii(40) As Integer
        Dim pass2 As String
        Dim lenp As Integer
        Dim i, j, k As Integer
        lenp = Len(pass)
        i = 1
        While i <= lenp
            pass1(i) = Mid(pass, i, 1)
            ascii(i) = Asc(pass1(i))
            ascii(i) = ascii(i) - (i + (i - 4))
            pass2 = pass2 & Chr(ascii(i))
            i = i + 1
        End While
        decrypt_pass = pass2
    End Function
End Module

File: JewelBox C#.NET/modMain.cs
using System.Diagnostics;
using Microsoft.VisualBasic;
using System;
using System.Data;
using System.Drawing;
using System.Windows.Forms;
using System.Collections;

namespace JewelBox2004
{
	sealed class modMain
	{
		
		public static int RowsAffected;
		public static int index;
		public static int numVals;
		public static int numRows;
		public static int x;
		public static string te1;
		public static string te2;
		public static string te3;
		public static string strConn = "Provider = SQLOLEDB;data source=DREAMWORKS;initial catalog=JewelBox;persist security info=False;user id=admin;password = velocity;workstation id=DREAMWORKS;packet size=4096";
		public static int repMod;
		public static string mytime;
		public static string loguser;
		public static string logpass;
		public static bool logsuccess;
		public static string sql;
		public static System.Data.SqlClient.SqlDataAdapter adapt = new System.Data.SqlClient.SqlDataAdapter();
		public static System.Data.SqlClient.SqlConnection conn = new System.Data.SqlClient.SqlConnection();
		public static System.Data.SqlClient.SqlCommand comm = new System.Data.SqlClient.SqlCommand();
		public static DataSet rslogin = new DataSet();
		public static string connString;
		
		public static void ConnectDB ()
		{
			connString = "initial catalog=JewelBox;password=delta123;persist security info=True;user id=sa;workstation id=GOLDENTIGER;packet size=4096";
			conn = new System.Data.SqlClient.SqlConnection(connString);
			comm = new System.Data.SqlClient.SqlCommand(connString, conn);
			comm.CommandType = CommandType.Text;
			try
			{
				conn.Open();
				Debug.WriteLine("Connection Success");
			}
			catch (System.Data.SqlClient.SqlException SqlExcep)
			{
				Debug.WriteLine(SqlExcep.Message);
			}
		}
		public static void commitDB ()
		{
			conn.Close();
			conn.Dispose();
			conn = null;
			ConnectDB();
		}
		
		public static void closeDB ()
		{
			
		}
		
		public static void Main ()
		{
			frmSplash frm = new frmSplash();
			frm.ShowDialog();
			ConnectDB();
		}
		
		public static string encrypt_pass(string pass)
		{
			string returnValue;
			string[] pass1 = new string[41];
			string[] ascii = new string[41];
			string pass2 = "";
			int lenp;
			int i;
			lenp = Strings.Len(pass);
			i = 0;
			while (i < lenp)
			{
				i = i + 1;
				pass1[i] = Strings.Mid(pass, i, 1);
				ascii[i] = System.Convert.ToString(Strings.Asc(pass1[i]));
				ascii[i] = ascii[i] +(i +(i - 4));
				pass2 = pass2 + Strings.Chr(System.Convert.ToInt32(ascii[i]));
			}
			returnValue = pass2;
			return returnValue;
		}
		public static string decrypt_pass(string pass)
		{
			string returnValue;
			string[] pass1 = new string[41];
			int[] ascii = new int[41];
			string pass2 = "";
			int lenp;
			int i;
			//			int j;
			//			int k;
			lenp = Strings.Len(pass);
			i = 1;
			while (i <= lenp)
			{
				pass1[i] = Strings.Mid(pass, i, 1);
				ascii[i] = Strings.Asc(pass1[i]);
				ascii[i] = ascii[i] -(i +(i - 4));
				pass2 = pass2 + Strings.Chr(ascii[i]);
				i = i + 1;
			}
			returnValue = pass2;
			return returnValue;
		}
	}
	
}
